src/customFunctions/customFetch.ts

```diff
@@ -0,0 +1,16 @@
+import { bearerTokenAuthorization } from "./../configs/bearerTokenAuthorization";
+
+export const customFetch = async (
+  input: RequestInfo,
+  init: RequestInit = {}
+) => {
+  init.headers = bearerTokenAuthorization.headers;
+  const response = await fetch(input, init);
+
+  if (response.ok) {
+    return {
+      json: await response.json(),
+      status: response.status,
+    };
+  } else throw { status: response.status };
```

Comment:
throwovat object se podle mě nedělá, nebo aspoň já na to nejsem zvyklej... když už něco throwovat, tak instanci erroru ne? 

navíc klienti customFetche pak nemají přístup k ničemu jinému, než ke statusu, takže přijdou o datovou informaci